integration-tests/baas-test-server/README.md

```diff
@@ -14,7 +14,13 @@ AWS_ACCESS_KEY_ID=...
 AWS_SECRET_ACCESS_KEY=...
 ```
 
-Be sure to have `assisted_agg` (if on Mac) or `libmongo.so` (if on Linux) downloaded and added to your PATH. More info about where to get it can be found [in the BAAS repo](https://github.com/10gen/baas/blob/master/etc/docs/onboarding.md).
+A file called `assisted_agg` needs to be downloaded in order for MongoDB aggregation tests to run:
+1. Download the file `assisted_agg` (if on Mac) or `libmongo.so` (if on Linux) and add it
+   to your `PATH` (see [BaaS Local Environment Setup](https://github.com/10gen/baas/blob/master/etc/docs/onboarding.md)).
```

Comment:
Should we just link to https://stitch-artifacts.s3.amazonaws.com/stitch-mongo-libs/stitch_mongo_libs_osx_patch_df9a68d900d7faddcee16bf0f532437da815a1c3_63289ac95623436dd66f7056_22_09_19_16_37_32/assisted_agg

Reply:
Not sure if we could make that public like that? The baas repo does not seem to be public, so linking to their repo would at least block people w/o access.

If not, could also link to the more specific section? (https://github.com/10gen/baas/blob/master/etc/docs/onboarding.md#third-party-mongo-libraries)

Reply:
Good point!  Then we should link to the specific section and indicate in the steps that the `assisted_agg` is available there.